packages/frontend/src/mobx-transient.ts

```diff
@@ -0,0 +1,124 @@
+import { BaseTransient, App } from '@h4bff/core';
+import { autorun, observable } from 'mobx';
+import { useLocalStore } from 'mobx-react-lite';
```

Comment:
Shouldn't this be imported from `mobx-react`?

Reply:
ah ok